app/lib/account.ts

```diff
@@ -14,10 +14,11 @@ export interface AccountResponse {
 
 export const account = async (
   identifier: string,
-  token: string | undefined
+  token: string | undefined,
+  basePath: string
 ): Promise<AccountResponse | { error: string }> => {
   try {
-    const res = await fetch(`http://localhost:3000/v0/accounts/${identifier}`, {
+    const res = await fetch(`${basePath}/v0/accounts/${identifier}`, {
```

Comment:
```suggestion
    const res = await fetch(new URL(`/v0/accounts/${identifier}`, basePath), {
```
パスと基底 URL を結合するときは, [`new URL` コンストラクタ](https://developer.mozilla.org/ja/docs/Web/API/URL/URL) を利用すると安全です. 他のパス結合をしている箇所も同様です.